.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+install:
+  - pip install flake8 docutils
+script:
+  - flake8 ckanharvester.py
+  - rst2html --strict README.rst /dev/null
```

Comment:
Sorry, my mistake, `rst2html` should be replaced to `rst2html.py`. See https://travis-ci.org/ivpk/opendata.gov.lt-mysql-import/builds/276750782#L526